README.md

```diff
@@ -170,7 +170,26 @@ We have implemented a number of commands in the `nf-core/tools` package to make
 2. Install [`nextflow`](https://nf-co.re/usage/installation) (`>=20.11.0-edge`; see [Nextflow edge releases](#nextflow-edge-releases))
 3. Install any of [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) or [`Conda`](https://conda.io/miniconda.html)
 4. [Fork and clone this repo locally](#uploading-to-nf-coremodules)
-5. Create a module using the [nf-core DSL2 module template](https://github.com/nf-core/tools/blob/master/nf_core/module-template/software/main.nf):
+5. Set up git by adding a new remote of the nf-core git repo called `upstream`
+
+   ```bash
+   git remote add upstream https://github.com/nf-core/modules.git
+   ```
+
+   Make a new branch for your module and check it out
+
+   ```bash
+   git checkout -b fastqc
+   ```
```

Comment:
I think I would move this to a step 6, as I think of it as a separate part of my workflow

Reply:
Wouldn't you want to make the branch first before running the create command and making the new file templates?

Reply:
It shouldn't matter because the files aren't being overwritten by the checkout of the branch, so @ewels method would work also. Just personal preference for how people like to work.

I think the focus of this is on new users. I agree with @aunderwo that creating a new branch before starting to work is easier to conceptualize.